ts/modules/timeline.src.ts

```diff
@@ -679,11 +681,11 @@ seriesType<Highcharts.TimelineSeries>('timeline', 'line',
                     pointMarkerOptions.width,
                     seriesMarkerOptions.width,
                     series.closestPointRangePx
-                ),
+                ) as any,
```

Comment:
Here the arguments are the problem, not the pick return. Or the pick typing is not working properly.